src/sqlparser/tests/testdata/select.yaml

```diff
@@ -37,6 +37,8 @@
   formatted_sql: SELECT id, fname, lname FROM customer WHERE salary <> 'Not Provided' AND salary <> ''
 - input: SELECT id FROM customer WHERE NOT salary = ''
   formatted_sql: SELECT id FROM customer WHERE NOT salary = ''
+- input: SELECT * EXCEPT v1 FROM foo
+  formatted_sql: SELECT EXCEPT v1 FROM foo
```

Comment:
Is this valid?

Reply:
This is valid in current implementation and there are many things need to be fixed and there will be possible change later.